FV3/conf/modules.fv3.stellar

```diff
@@ -0,0 +1,29 @@
+# NOTE: the "module purge" and loading of the module command are
+# handled by the module-setup.sh (or .csh) script.
+##
+## load programming environment
+## this typically includes compiler, MPI and job scheduler
+##
+
+# Kai's modules (note these are commented out in favor of the more up-to-date versions):
```

Comment:
In formalizing this, I think it would be good to include Kai's full name here:
```suggestion
# Modules suggested by Kai-Yuan Cheng.  Note these are commented out in favor
# of the more up-to-date versions):
```